docs/upgrade_guide_cloud.md

```diff
@@ -0,0 +1,204 @@
+```bash
+CREATE DATABASE "orch-infra-rps";
+REVOKE CREATE ON SCHEMA public FROM PUBLIC;
+REVOKE ALL ON DATABASE "orch-infra-rps" FROM PUBLIC;
+CREATE USER "orch-infra-rps-user" WITH PASSWORD '<USER_DEFINED_PASSWORD>';
+GRANT CONNECT ON DATABASE "orch-infra-rps" TO "orch-infra-rps-user";
+GRANT ALL PRIVILEGES ON DATABASE "orch-infra-rps" TO "orch-infra-rps-user";
+ALTER DATABASE "orch-infra-rps" OWNER TO "orch-infra-rps-user";
+```
+
+3. Create orch-infra-mps DB and user
+```bash
+CREATE DATABASE "orch-infra-mps";
+REVOKE CREATE ON SCHEMA public FROM PUBLIC;
+REVOKE ALL ON DATABASE "orch-infra-mps" FROM PUBLIC;
+CREATE USER "orch-infra-mps-user" WITH PASSWORD '<USER_DEFINED_PASSWORD>';
+GRANT CONNECT ON DATABASE "orch-infra-mps" TO "orch-infra-mps-user";
+GRANT ALL PRIVILEGES ON DATABASE "orch-infra-mps" TO "orch-infra-mps-user";
+ALTER DATABASE "orch-infra-mps" OWNER TO "orch-infra-mps-user";
+\q
+```
+> **⚠️ Note**
+> <USER_DEFINED_PASSWORD> This is the password of your choice. Use the same password everywhere where this appears.
+
+4. Create kubernetes secrets for DB users created above
+```bash
+kubectl create secret generic mps-aurora-postgresql \
+  --from-literal=PGDATABASE=orch-infra-mps \
+  --from-literal=PGHOST=<host-endpoint-url> \
+  --from-literal=PGPASSWORD=<USER_DEFINED_PASSWORD> \
+  --from-literal=PGPORT=5432 \
+  --from-literal=PGUSER=orch-infra-mps-user \
+  --from-literal=password=<USER_DEFINED_PASSWORD> \
+  -n orch-infra
+```
+
+```bash
+kubectl create secret generic mps-reader-aurora-postgresql \
+  --from-literal=PGDATABASE=orch-infra-mps \
+  --from-literal=PGHOST=<host-endpoint-url> \
+  --from-literal=PGPASSWORD=<USER_DEFINED_PASSWORD> \
+  --from-literal=PGPORT=5432 \
+  --from-literal=PGUSER=orch-infra-mps-user \
+ --from-literal=password=<USER_DEFINED_PASSWORD> \
+  -n orch-infra
+```
+
+```bash
+kubectl create secret generic rps-aurora-postgresql \
+  --from-literal=PGDATABASE=orch-infra-rps \
+  --from-literal=PGHOST=<host-endpoint-url> \
+  --from-literal=PGPASSWORD=<USER_DEFINED_PASSWORD> \
+  --from-literal=PGPORT=5432 \
+  --from-literal=PGUSER=orch-infra-rps-user \
+ --from-literal=password=<USER_DEFINED_PASSWORD> \
+  -n orch-infra
+```
+
+```bash
+kubectl create secret generic rps-reader-aurora-postgresql \
+  --from-literal=PGDATABASE=orch-infra-rps \
+  --from-literal=PGHOST=<host-endpoint-url> \
+  --from-literal=PGPASSWORD=<USER_DEFINED_PASSWORD> \
+  --from-literal=PGPORT=5432 \
+  --from-literal=PGUSER=orch-infra-rps-user \
+ --from-literal=password=<USER_DEFINED_PASSWORD> \
+  -n orch-infra
+```
+
+### Step 3: Delete/resync amt-dbpassword-secret-job pod in infra-extermal app
+Once DB, user and secrets creation is done, you need to delete/resync amt-dbpassword-secret-job pod and it should make infra-external app healthy.
+
+
+### Step 4: Verification
+Log into web UI of the orchestrator.
+Go to Settings->OS profiles. There you should see the any of the toolkit version upgraded to latest.
```

Comment:
Add post upgrade steps / notes to onboard nodes, create clusters and deploy apps